app/assets/javascripts/discourse/public/assets/scripts/discourse-test-load-dynamic-js.js

```diff
@@ -0,0 +1,28 @@
+const dynamicJsTemplate = document.querySelector("#dynamic-test-js");
+
+const params = new URLSearchParams(document.location.search);
+const skipPlugins = params.get("qunit_skip_plugins");
+
+for (const element of Array.from(dynamicJsTemplate.content.childNodes)) {
```

Comment:
```suggestion
for (const element of dynamicJsTemplate.content.childNodes) {
```

That is always iterable imo

Reply:
Right! My initial implementation was to remove nodes from the dynamicJsTemplate rather than cloning them, so simple iteration wouldn't work because the iterable items were changing during the loop.

But now that I switched to the clone approach, `for...of` works perfectly - thanks! 👌